server/controllers/class/getAnnouncement.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response } from 'express';
+import getAnnouncementQuery from '../../queries/class';
+
+const getAnnouncement = async (req:Request, res:Response) => {
+  const { classId } = req.params;
+
+  const data:any = await getAnnouncementQuery(classId);
+  res.json(data);
+};
```

Comment:
what if an error happens in getAnnouncementQuery? Add your code in try and catch so you can catch the error.